src/core/utils/hooks/useWindowSize.tsx

```diff
@@ -0,0 +1,28 @@
+import {useState, useEffect} from "react";
```

Comment:
> using useWindowSize (this will be removed, only there for testing purposes).

I saw this part, but this hook is not actually used at the moment, so maybe you can remove from remote, but keep it on your local if you need it

We could add an example of changing `items` to the story, but I don't think that's necessary

Reply:
I added that for testing purposes, it will be removed before the pr is merged